client/my-sites/earn/refer-a-friend/style.scss

```diff
@@ -0,0 +1,11 @@
+.refer-a-friend__earn-page {
+	.promo-section__promos {
+		.promo-card {
+			width: calc( 100% - 1em );
+
+			@include breakpoint( '>1280px' ) {
+				width: calc( 100% - 1em );
```

Comment:
Is this breakpoint needed if it's just redefining the same style as the default?

Reply:
It overrides the default style for the promo cards, which is 50% at this breakpoint.